runtime/vm/jvminit.c

```diff
@@ -2847,9 +2926,12 @@ VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved)
 		case SYSTEM_CLASSLOADER_SET :
 
 			loadInfo = FIND_DLL_TABLE_ENTRY( FUNCTION_VM_INIT );
-			if (NULL == (vm->systemClassLoader = allocateClassLoader(vm))) {
-				setErrorJ9dll(PORTLIB, loadInfo, "cannot allocate system classloader", FALSE);
-				goto _error;
+			/* classLoader is already set during the restore run. */
```

Comment:
nit: This comment needs work. There's nothing nearby (that I can see) called "classLoader"; it should say `systemClassLoader` or "The system class loader".